### ISSUE TITLE
E6E Starmetal Automation

### ISSUE THREAD

**Issue by Kevin-Marsh** [EnigmaticaModpacks/Enigmatica6#4286]
### Describe your suggestion

At the moment you can get starmetal blocks in the starlight infuser by using a celestial honeycomb block and maybe starlight to 3 blocks of starmetal. This is an awesome way to get starmetal but not automatable since it doesn't let you use a fake player to use the resonating wand to activate the change.

I suggest adding a celestial honeycomb and a full bucket of starlight into the thermopneumatic press for 1 block starmetal. This would make it more expensive than the manual method with a definite chance of consuming the starlight and only producing 1 block but would allow for full lategame automation of starmetal.

### Why would you like this added/changed?

Will make it easier to automate starmetal in the lategame.

**Comment by telemenar** [comment]
You can already automate it though by just making the ore and smelting?

**Comment by MuteTiefling** [comment]
So, thinking back on this... I believe we added this recipe specifically because it couldn't be automated as it's very generous. @TemperedFool does that sound right to you?

The automated way would be to set up block placers/breakers and process the resulting ore as best you can.

**Comment by TemperedFool** [comment]
Yeah, it wouldn't be even close to this generous if it were automatable.

**Comment by TemperedFool** [comment]
This really seems like a solution in search of a problem. I automated this with 2 routers and some chunk loading...
